### PR TITLE
added editable property to lane object

### PR DESCRIPTION
**Pull request by ahmetcetin**
it gives possibility to mark some lanes as not editable if board is set editable

### CODE REVIEW

File: src/components/Lane.js [hunk 1]
```diff
@@ -204,7 +205,7 @@ class Lane extends Component {
           getChildPayload={index => this.props.getCardDetails(id, index)}>
           {cardList}
         </Container>
-        {editable && !addCardMode && this.renderAddCardLink()}
+        {editable && selfEditable && !addCardMode && this.renderAddCardLink()}
```
Comment (G-Rath) [inline src/components/Lane.js]: There's no need for this to be `selfEditable` - instead, just call it `editable`, and refactor the current `editable` *variable* to be `boardEditable` (the prop can stay the same - it's just the local variable name that needs to change).

Comment (ahmetcetin) [reply src/components/Lane.js]: done

**Comment by ahmetcetin** [comment]
change request implemented

**Comment by oflarcade** [comment]
any update on the conflict for editable lanes ?

File: src/components/BoardContainer.js [hunk 2]
```diff
@@ -145,6 +145,9 @@ class BoardContainer extends Component {
       'newLaneTemplate',
       'newCardTemplate'
     ])
+    
+    passthroughProps.boardEditable = passthroughProps.editable
+    delete passthroughProps.editable
```
Comment (dapi) [inline src/components/BoardContainer.js]: Looks weird. It there any functional way to do same?

Comment (dapi) [reply src/components/BoardContainer.js]: immutability-helper?

**Comment by dapi** [comment]
Do I get it right? Do you want to individually disable/enable availability to add a new card into lanes?

**Comment by dapi** [comment]
@ahmetcetin hi!

Use `lane.disallowAddingCard` - https://github.com/rcdexta/react-trello/pull/287/files#diff-e2b1352494cba6252bc14efc8ccc5ac0R178

**Comment by dapi** [comment]
Thank you for idea and PR!

**Pull request closed without merging**